src/Components/Links.js

```diff
@@ -8,6 +8,7 @@ function Links({ links }) {
     youtube: 'red',
     twitter: 'blue',
     github: 'black',
+    linkdin: 'blue'
```

Comment:
```suggestion
    linkedin: 'blue',
```

Fix the lint issue so the build passes, and correct spelling mistake.